cachalot/tests/read.py

```diff
@@ -352,7 +352,7 @@ def test_many_to_many_when_no_sql_check(self):
     @all_final_sql_checks
     def test_subquery(self):
         additional_tables = []
-        if django_version[0] >= 4 and settings.CACHALOT_FINAL_SQL_CHECK:
+        if django_version[0] >= 4 and django_version[1] < 1 and settings.CACHALOT_FINAL_SQL_CHECK:
```

Comment:
this would also pass for django 5.0

Reply:
I changed it so it's looking for 4.0.x only